ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -54,14 +58,21 @@ internal fun Footer(
     childModifier: Modifier = Modifier,
     allPlansTapped: (() -> Unit)? = null,
 ) {
-    Footer(
-        mode = templateConfiguration.mode,
-        configuration = templateConfiguration.configuration,
-        colors = templateConfiguration.getCurrentColors(),
-        viewModel = viewModel,
-        childModifier = childModifier,
-        allPlansTapped = allPlansTapped,
-    )
+    Column {
+        Footer(
+            mode = templateConfiguration.mode,
+            configuration = templateConfiguration.configuration,
+            colors = templateConfiguration.getCurrentColors(),
+            viewModel = viewModel,
+            childModifier = childModifier,
+            allPlansTapped = allPlansTapped,
+        )
+        Spacer(
```

Comment:
Hmm I'm concerned this could cause problems if the footer has a background color and we apply the spacer without a background color...

Reply:
I don't think we are applying a color to the background of the footer, so that shouldn't be an issue atm? I tried adding 

```
.windowInsetsBottomHeight(
                WindowInsets.systemBars,
            )
        ```
        
        to the Row in the Footer but that doesn't work properly

Reply:
Ohh wait 🤦‍♂️ I didn't remember this was only for the buttons at the bottom. Ignore this then! Sorry about the noise!